lit/docs/auth/configuring/generic-saml.lit

```diff
@@ -0,0 +1,101 @@
+\title{Generic SAML auth}{generic-saml-auth}
+
+\use-plugin{concourse-docs}
+\omit-children-from-table-of-contents
+
+A Concourse server can authenticate against any valid SAML auth provider.
+
+\section{
+  \title{Authentication}{generic-saml-authentication}
+
+  First you'll need to create a client with your SAML provider.
+
+  The SAML Assertion Consumer Service (ACS) URL and the Audince URI (SP Entity ID)
+  must be the URL of your Concourse server with \code{/sky/issuer/callback} appended.
+  This address should be reachable by Concourse users - can be an internal URL
+  and does not have to be publicly reachable.
+
+  For example, Concourse's own CI server's callback URL would be:
+
+  \codeblock{bash}{{{
+  https://ci.concourse-ci.org/sky/issuer/callback
+  }}}
+
+  Attribute statments that you define in the SAML provider need to be mapped in concourse:
+
+  \codeblock{bash}{{{
+  CONCOURSE_SAML_USERNAME_ATTR=name   # default
+  CONCOURSE_SAML_EMAIL_ATTR=email     # default
+  CONCOURSE_SAML_GROUPS_ATTR=groups   # default
+  }}}
+
+  Finally, the SAML provider will generate a SSO URL and a CA certificate required by Concourse.
```

Comment:
Maybe also worth mentioning that it gives an `Identity Provider Issuer` that corresponds with `CONCOURSE_SAML_SSO_ISSUER` - I'm not sure whether is an added layer of security in SAML, but at the very least it's probably a good idea to mention (since I was confused why I didn't need to use it despite Okta telling me to)